docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.2'
+
+services:
+  test:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    volumes:
+      - ./packages/react/cypress-visual-screenshots:/carbon-addons-iot-react/cypress-visual-screenshots
+      - ./packages/react/cypress-visual-report:/carbon-addons-iot-react/cypress-visual-report
```

Comment:
share the `packages/react/cypress-visual-*` folders, so that the images generated in the container are stored in the host filesystem and can be committed into git.